### PR TITLE
Checking end date.

### PR DESCRIPTION
**Pull request by Aditi-0725**
End Date should always be 7 days ahead of the Start Date.

### CODE REVIEW

**Review by Devashish0210 — COMMENTED** [review]
Looks good to me. Good job 👍

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (swarajpure) [inline .vscode/launch.json:15]: Please delete this file, not needed on production

File: script.js [hunk 2]
```diff
@@ -38,6 +38,33 @@ const showSubmitLoader = (show = true) => {
...
+
+document.getElementById("endsOn").addEventListener("change", (event) => {
+  if (event.target.value) {
+    if (startedDate.value < endDate.value) {
+      alert("End Date should be greater than the Start Date")
+    }
+  }
+})
+
+document.getElementById("startedOn").addEventListener("change", function (event) {
```
Comment (swarajpure) [inline script.js:62]: We've declared startedDate and endDate but we're not using it on line 51, 54, 62... 🤔

File: script.js [hunk 3]
```diff
@@ -161,5 +188,6 @@ const dd = String(currentDate.getDate()).padStart(2, '0');
 const mm = String(currentDate.getMonth() + 1).padStart(2, '0');
 const yyyy = currentDate.getFullYear();
 
-today = `${yyyy}-${mm}-${dd}`;
-document.getElementById('startedOn').value = today;
+today = `${yyyy}-${mm}-${dd}`;
```
Comment (swarajpure) [inline script.js:191]: Have we declared `today` as `const` or `let` somewhere?

File: script.js [hunk 4]
```diff
@@ -161,5 +188,6 @@ const dd = String(currentDate.getDate()).padStart(2, '0');
 const mm = String(currentDate.getMonth() + 1).padStart(2, '0');
 const yyyy = currentDate.getFullYear();
 
-today = `${yyyy}-${mm}-${dd}`;
-document.getElementById('startedOn').value = today;
+today = `${yyyy}-${mm}-${dd}`; 
+document.getElementById("startedOn").value = today;
```
Comment (swarajpure) [inline script.js:192]: Again, we can use `startedDate` here

**Review by swarajpure — CHANGES_REQUESTED** [review]
Hope this PR taught you a lot 🥳 

Suggesting some changes, please check! 🙂

File: script.js (hunk 3 above)
Comment (Aditi-0725) [reply script.js:191]: no

Comment (swarajpure) [reply script.js:191]: Let us do that please :)
Let's initialize variables and use

Comment (Aditi-0725) [reply script.js:191]: okay

File: script.js (hunk 4 above)
Comment (Aditi-0725) [reply script.js:192]: yes, i've made the changes.

File: script.js (hunk 3 above)
Comment (Aditi-0725) [reply script.js]: yes, i have made the changes.

File: script.js [hunk 5]
```diff
@@ -161,5 +188,6 @@ const dd = String(currentDate.getDate()).padStart(2, '0');
 const mm = String(currentDate.getMonth() + 1).padStart(2, '0');
 const yyyy = currentDate.getFullYear();
 
-today = `${yyyy}-${mm}-${dd}`;
-document.getElementById('startedOn').value = today;
+let today = `${yyyy}-${mm}-${dd}`;
```
Comment (swarajpure) [inline script.js:191]: is `today` being changed later? Why are we not using `const` here?

Comment (Aditi-0725) [reply script.js:191]: i declared it as let just to be on the safer side.

**Comment by Aditi-0725** [comment]
> Looks good to me. Good job 👍

Thank you Devashish!

**Comment by Aditi-0725** [comment]
> Hope this PR taught you a lot 🥳
> Yes!! And thank you so much Swaraj for helping me. 
> Suggesting some changes, please check! 🙂

**Comment by Aditi-0725** [comment]
> Hope this PR taught you a lot 🥳
> 
> Suggesting some changes, please check! 🙂

Done with all the changes.

File: script.js (hunk 5 above)
Comment (swarajpure) [reply script.js:191]: If we know that the variable value isn't going to change, let's keep it `const` please

**Pull request closed without merging**